### DISCUSSION TITLE
v2.02-b7没有任何图标 打开客户端disconnected

### DISCUSSION THREAD

**Discussion by zrd007** [cangzhang/ChampR#285] [General]
b6版本没什么问题

**Comment by zrd007** [comment]
![image](https://github.com/cangzhang/champ-r/assets/68060099/ca2ad8bc-98ad-49a4-afc8-4a854c2ed6aa)

**Comment by cangzhang** [comment]
服务器被你的 isp 屏蔽了，把 `ql-rs.lbj.moe` 加到代理规则里

**Reply by LYLYILY** [reply]
你好，我想问一下怎么添加代理规则

**Reply by cangzhang** [reply]
如果是 clash 可以根据这个教程来 https://docs.gtk.pw/contents/ui/profiles/rules.html
简单来说就是：
```
DOMAIN,ql-rs.lbj.moe,你的代理
```

**Reply by LYLYILY** [reply]
好的，谢谢
